src/pybind/mgr/cephadm/templates/services/loki.yml.j2

```diff
@@ -0,0 +1,27 @@
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+  grpc_listen_port: 8080
```

Comment:
Shouldn't these ports somehow read from the other places where defined (I count 9 other places). We should, as much as possible, try to reduce harcoding values (ports, paths, IPs, etc.), and read them from a single source of truth.

For the time being these ports cannot be changed, but I'd be surprised that no one had already complained about the inability to configure service ports.